services/app-api/forms/mcpar.json

```diff
@@ -2689,8 +2752,8 @@
               "spreadsheet": "D2_Plan_Measures",
               "info": [
                 {
-                  "type": "html",
-                  "content": "<p>Report on individual measures in each of the following eight domains: (1) Primary care access and preventive care, (2) Maternal and perinatal health, (3) Care of acute and chronic conditions, (4) Behavioral health care, (5) Dental and oral health services, (6) Health plan enrollee experience of care, (7) Long-term services and supports, and (8) Other. For composite measures, be sure to include each individual sub-measure component.</p>"
+                  "type": "p",
```

Comment:
Does this one not need the `children` prop like the ones above?

Reply:
I believe the `children` prop is just for when there are multiple elements included in the paragraph, like when we have an embedded link. If it's just a single chunk of text with no additional elements, it doesn't need the `children`

Reply:
Ah gotcha.